pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -398,64 +398,6 @@ func TestEnoughRequests(t *testing.T) {
 	}
 }
 
-func TestPreFilterDisabled(t *testing.T) {
```

Comment:
We should keep this test and verify that an error is returned when invoking Filter without calling PreFilter. The unit test should be simple.

Reply:
Done